src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs

```diff
@@ -421,6 +421,18 @@ public static string GetMSBuildFilePathForPackageReferenceStyleProject(PackageSp
             return Path.Combine(project.RestoreMetadata.OutputPath, $"{projFileName}.nuget.g{extension}");
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="project"></param>
```

Comment:
Not sure what the value is of a description that doesn't really add anything.

Reply:
I see lots of 👎 by @donnie-msft here. 

`<summary></summary>` is the same as having nothing to the intellisense documentation.

Reply:
That's correct, but the template generates all the parameters. Comments are not compiled. Our codebase is full of empty parameters. If you have a proposal for a Coding Guideline, we should discuss that separately from this PR. This PR is following existing conventions.

Reply:
Added some more docs.